doc/psidk/src/development/services/rust-service/package.md

```diff
@@ -10,6 +10,12 @@ In the top-level crate:
 
 - `package-name`: Required.
 - `services`: A list of services to include in the psibase package.
+- `server`: May be present on any crate that builds a service. The value is a crate which will handle HTTP requests sent to this service. The other crate will be built and included in the current package. If HTTP requests are handled in the base service, it should reference the service. Can be omitted if service serves no HTTP requests.
```

Comment:
These don't seem to belong here in the "In the top-level crate" section

Reply:
This looks like a bad merge